gulpfile.js

```diff
@@ -153,7 +154,7 @@ gulp.task( 'styles', function() {
  *     4. Uglifes/Minifies the JS file and generates vendors.min.js
  */
 gulp.task( 'vendorsJS', function() {
-	return gulp.src( config.jsVendorSRC )
+	return gulp.src( config.jsVendorSRC, {since: gulp.lastRun( 'vendorsJS' ) } ) // Only run on changed files
```

Comment:
Always add a `.` at the end of a comment if it doesn't end with some sort of code and start it with capital case. Just add the periods and I'll merge it. Everything else looks good.

Reply:
Fixed